src/BudgetManager.Shared/BudgetManager.Shared.Models.MongoDB/Interfaces/IModelBase.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BudgetManager.Shared.Models.MongoDB.Models.Interfaces
+{
+    public interface IModelBase
+    {
+        Guid Id { get; set; }
```

Comment:
May be it would be better to make in such way:
```suggestion
    public interface IModelBase<T>
    {
        T Id { get; set; }
```
 Just to make our code more common and reusable

Reply:
Since every non-common model has to implement the interface now, I am not sure if it is a good idea because we will need to specify Id type on every single model.